site_scons/site_tools/go_builder.py

```diff
@@ -49,6 +49,20 @@ def _scan_go_file(node, env, _path):
     return includes
 
 
+def get_go_version(output):
+    """Capture only the number, ignoring optional rc"""
+    one_part = '([0-9]+)(?:rc[0-9]+)?'
+    return '.'.join(re.findall(fr'(?:go)?{one_part}\.{one_part}\.{one_part}', output)[0])
```

Comment:
```suggestion
"""Capture only the version after 'go'"""
ver_re = re.compile(r'go([0-9\.]+)')
groups = ver_re.findall(output)
if not groups or len(groups) == 0:
    return ""
return groups[0]
```

Reply:
Done.  I used None rather than "" but the regexp is what you suggested.